frontend/public/style/_icons.scss

```diff
@@ -6,6 +6,14 @@
   color: $color-red-error;
 }
 
+.route-accepted {
```

Comment:
Is the `$color-green-success` different than the color `text-success` provides?   And if so, there isn't an existing `text-` class that provides this color?  Just curious if we need special `route-` classes for the color change.

Reply:
Meaning if it is supposed to be consistent with `node`, per the JIRA, perhaps a generic class helps ensure future consistency if the pattern continues elsewhere.

Reply:
Hi, updated so route & node are using same css, pls see screenshots below.  -thanks